src/sample_weight_audit/estimator_check.py

```diff
@@ -163,7 +170,7 @@ def get_cv_params(
     return extra_params_weighted, extra_params_repeated
 
 
-def compute_predictions(est, X):
+def compute_predictions(est, X, y=None, score=True):
```

Comment:
This is no longer computing predictions. We need to always adjust function names and variable names to reflect what the code is doing, otherwise the code will quickly become unmaintainable.

If we want to keep the `score=False` option, let's instead write 2 functions: `compute_predictions(est, X)` and another called `score_estimator(est, X, y)` and the latter can call the former internally to avoid code duplication if needed.